tla-types/src/test/scala/at/forsyte/apalache/tla/typecomp/TestHybrid.scala

```diff
@@ -55,131 +57,164 @@ class TestHybrid extends BuilderTest {
 
   test("decl") {
 
-    type T = TBuilderOperDeclInstruction
+    type ExplicitT = (String, TBuilderInstruction, Seq[TypedParam])
     type TParam = (TlaType1, Seq[TlaType1])
 
-    // A(p1,...,pn) == CHOOSE x: p1 /\  p2 >= 0 /\ ... /\ pn = pn
-    def mkWellTyped(tparam: TParam): (T, T) = {
+    // We create a different term for operator vs simple parameters
+    def paramCond(param: TypedParam): TBuilderInstruction = {
```

Comment:
This needs more explanation.
Apparently it takes operators and simple parameters, but I have no idea what it's supposed to return or how it's supposed to be used.